### PR TITLE
Suggest better code styles using dark background

### PR DESCRIPTION
**Pull request by CPatchane**
Use darker code blocks could be great since the theme is light. It uses extra_css file and it could be more improved.

screen shot 2018-01-26 at 01 05 32

### CODE REVIEW

**Comment by CPatchane** [comment]
@enguerran True! It's done 👍

**Comment by CPatchane** [comment]
@clochix I didn't built the docs in the last PR (maybe why you didn't see the changes). I push a build in this one so. Let me know if this is the process or not :)

File: mkdocs.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ repo_url: https://github.com/cozy/cozy-docs-v3
 edit_uri: edit/master/src/
 site_description: Cozy documentation
 site_author: CozyCloud - https://cozy.io
-extra_css: [extra.css]
+extra_css: [css/extra.css, css/highlight_theme.css]
```
Comment (clochix) [inline mkdocs.yml]: Could you please do the same for `mkdocs_fr.yml`, which is used to build the french flavor ?

**Comment by clochix** [comment]
@enguerran is is OK for thou?

File: mkdocs.yml (hunk 1 above)
Comment (CPatchane) [reply mkdocs.yml]: Sure. It's done 👍

**Comment by enguerran** [comment]
I may miss something in @CPatchane 's message
> @enguerran True! It's done 👍

as I can see a lot of css code to maintain here https://github.com/cozy/cozy-docs-v3/pull/32/files#diff-6a953be49abf7a2f5d53b1c0934ad84c

Can't we use a modern approach to embed external libraries?

**Comment by CPatchane** [comment]
@enguerran Unfortunately I didn't find any other way than define our own styles for highlightjs to change that :/

**Comment by clochix** [comment]
Hello! sadly, we can’t spend too much time styling the documentation, so I will just merge and close, so we can work on more urgent tasks. Thanks to both of you.

**Pull request merged**